Source/diablo.cpp

```diff
@@ -1901,3 +1904,38 @@ void diablo_color_cyc_logic()
 		}
 	}
 }
+
+#ifdef HELLFIRE
+void alloc_plr()
+{
+	plr = get_plr_mem(NULL);
+
+	if(plr == NULL) {
+		app_fatal("Unable to initialize memory");

```

Comment:
Was wondering what this function did :) Let me just update the config and then I'll do a diff.